app/javascript/dashboard/components/helpCenter/Header/ArticleHeader.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+  <div class="header--wrap">
+    <div class="header-left--wrap">
+      <h3 class="page-title">{{ headerTitle }}</h3>
+      <span class="text-block-title count-view">{{ `(${count})` }}</span>
+    </div>
+    <div>
+      <woot-button
+        class-names="article--buttons"
+        icon="filter"
+        color-scheme="secondary"
+        variant="hollow"
+        size="small"
+        @click="openFilterModal"
+      >
+        {{ $t('HELP_CENTER.HEADER.FILTER') }}
+      </woot-button>
+      <woot-button
+        class-names="article--buttons"
+        icon="arrow-sort"
+        color-scheme="secondary"
+        size="small"
+        variant="hollow"
+        @click="openDropdown"
+      >
+        {{ $t('HELP_CENTER.HEADER.SORT') }}
+        <span class="selected-value">
+          {{ selectedValue }}
+          <Fluent-icon class="dropdown-arrow" icon="chevron-down" size="14" />
+        </span>
+      </woot-button>
+      <div
+        v-if="showSortByDropdown"
+        v-on-clickaway="closeDropdown"
+        class="dropdown-pane dropdown-pane--open"
+      >
+        <woot-dropdown-menu>
+          <woot-dropdown-item>
+            <woot-button
+              variant="clear"
+              color-scheme="secondary"
+              size="small"
+              icon="send-clock"
+            >
+              {{ 'Status' }}
+            </woot-button>
+          </woot-dropdown-item>
+          <woot-dropdown-item>
+            <woot-button
+              variant="clear"
+              color-scheme="secondary"
+              size="small"
+              icon="dual-screen-clock"
+            >
+              {{ 'Created' }}
+            </woot-button>
+          </woot-dropdown-item>
+          <woot-dropdown-item>
+            <woot-button
+              variant="clear"
+              color-scheme="secondary"
+              size="small"
+              icon="calendar-clock"
+            >
+              {{ 'Last edited' }}
+            </woot-button>
+          </woot-dropdown-item>
+        </woot-dropdown-menu>
+      </div>
+      <woot-button
+        v-tooltip.top-end="$t('HELP_CENTER.HEADER.SETTINGS_BUTTON')"
+        icon="settings"
+        class-names="article--buttons"
+        variant="hollow"
+        size="small"
+        color-scheme="secondary"
+      />
+      <woot-button
```

Comment:
Two minor things needs to be changed here.

1. This component doesn't need to accept the prop `buttonLabel`. we can remove that prop altogether 

2. The new article should be a `router link` instead of button.

Reply:
How can we use the `router-link` in the storybook? Showing some error.